Dockerfile.deb

```diff
@@ -0,0 +1,47 @@
+FROM debian:bullseye-slim
+
+ARG KONG_VERSION=2.8.0
+ENV KONG_VERSION $KONG_VERSION
+
+ARG KONG_SHA256="54a3079ccb9e3751bb096500b25d6d34bdcdf4894e3b34b84b8778a5f99e18d2"
+
+ARG ASSET=remote
+ARG EE_PORTS
+
+COPY kong.deb /tmp/kong.deb
```

Comment:
I noticed the `kong.rpm` and `kong.apk.tar.gz` assets in this PR are empty (placeholders, I assume) whereas `kong.deb` is not. Any reason for that or maybe an oversight?

Reply:
whoops it should be empty good catch :+1: